package.json

```diff
@@ -31,7 +31,6 @@
     "dist/**/*.js",
     "dist/**/*.d.ts"
   ],
-  "types": "index.d.ts",
```

Comment:
Why we remove this line?

Reply:
The path is incorrect. It should be `dist/index.d.ts`. And since the `main` is `dist/index.js`, TypeScript would automatically infer to `dist/index.d.ts`. So it is not needed.